src/components/Radio/radio.scss

```diff
@@ -0,0 +1,12 @@
+.formio-form {
```

Comment:
Is this for swe or for all?

Reply:
this is not swe related,
this is for all form usage even without swe, it's about adding a optional class `radio-tab` a consumer can choose to use or not

Reply:
all good then :)